src/main/java/org/jasig/portlet/newsreader/mvc/AbstractNewsController.java

```diff
@@ -50,16 +101,18 @@ public boolean isAdmin(PortletRequest req) {
     @ModelAttribute("isGuest")
     public boolean isGuest(PortletRequest req) {
         return req.getRemoteUser() == null;
-    }    
+    }
 
     @ModelAttribute("supportsEdit")
     public boolean supportsEdit(PortletRequest req) {
-        return req.isPortletModeAllowed(PortletMode.EDIT);
-    }    
+        final PortletPreferences prefs = req.getPreferences();
+        final String allowEdit = prefs.getValue(ALLOW_EDIT_PREFERENCE, "true");
+        return Boolean.parseBoolean(allowEdit);
+    }
```

Comment:
@jameswennmacher -- I meant to cover this point in the writeup and forgot:  this is a new major version, 4.0.0.  The pom has already been bumped -- with earlier changes -- but there has been no release yet.